padglobal/padglobal.py

```diff
@@ -735,6 +767,8 @@ def which_to_text(self):
                 continue
```

Comment:
This looks out of date I think you need to re-pull.

Reply:
done